core/controllers/base.py

```diff
@@ -143,9 +142,11 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
         # Initializes the return dict for the handlers.
         self.values = {}
 
-        self.user = current_user_services.get_current_user()
-        self.user_id = current_user_services.get_user_id(
-            self.user) if self.user else None
+        self.user_id = (
+            current_user_services.get_user_id(
```

Comment:
should I modify the `current_user_services.get_user_id()` so that it returns None if object is None or make a separate function ?